iroc_fleet_manager/CoverageMissionRobot[] robots 
mrs_msgs/Point2D[] search_area

